bluebrain/repo-bluebrain/packages/py-astrovascpy/package.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyAstrovascpy(PythonPackage):
+    """
+    Vasculature blood flow computation and impact of astrocytic
+    endfeet on vessels
+    """
+
+    homepage = "https://github.com/BlueBrain/AstroVascPy"
+
+    url = "https://github.com/BlueBrain/AstroVascPy/releases/tag/0.1"
+
+    maintainers("tristan0x")
+
+    version("0.1", sha256="1e9b2036e9c1ce4cb0650612c0be2e7e7b142754e1550dfbfb0b757cee4f8bdd")
+
+    variant("vtk", default=False, description="add VTK support (mainly for visualization)")
+
+    depends_on("py-setuptools", type=("build", "run"))
+    depends_on("py-setuptools-scm", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("py-setuptools-scm", type="build")
```
I recall this being a build dependency mostly (just saw that when I hit approve :facepalm:)

Reply:
Sorry, that comment may be contradictory to the other one. Pure Python build dependencies should really be `build` only.